Assets/MixedRealityToolkit/Services/MixedRealityToolkit.cs

```diff
@@ -324,7 +324,7 @@ private void InitializeServiceLocator()
             //If the Mixed Reality Toolkit is not configured, stop.
             if (ActiveProfile == null)
             {
-                Debug.LogError("No Mixed Reality Configuration Profile found, cannot initialize the Mixed Reality Toolkit");
+                Debug.LogWarning("No Mixed Reality Configuration Profile found, cannot initialize the Mixed Reality Toolkit");
```

Comment:
I think this should stay as an error. Having no active profile is not a state we should ever be in.

```suggestion
                Debug.LogError("No Mixed Reality Configuration Profile found, cannot initialize the Mixed Reality Toolkit");
```

Reply:
If this is happening because we're manually setting the profile to null for some reason, it should just be swapped directly to whichever new profile is going to be set.

If we see issues with that approach, pinging @davidkline-ms  :)

Reply:
I tend to agree, @keveleigh